src/app/modules/verification/verification.service.ts

```diff
@@ -455,7 +458,7 @@ const sendOtpForField = (
   switch (fieldType) {
     case BasicField.Mobile:
       return fieldId
-        ? FormService.retrieveFormById(formId)
+        ? FormService.retrieveFullFormById(formId)
```

Comment:
why do we need full form?

Reply:
`verifyUserBetaflag` requires `form.admin`

Reply:
This will be removed after we're completed the migration.